dace/codegen/targets/intel_fpga.py

```diff
@@ -753,7 +758,11 @@ def generate_nsdfg_arguments(self, sdfg, dfg, state, node):
                 continue
             desc = sdfg.arrays[in_memlet.data]
             ptrname = cpp.ptr(in_memlet.data, desc, sdfg, self._frame)
-            defined_type, defined_ctype = self._dispatcher.defined_vars.get(ptrname, 1)
+            defined_type, defined_ctype = self._dispatcher.defined_vars.get(ptrname, 1) #Ctype -> dacetype
```

Comment:
I didn't get the comment, could you explain it?

Reply:
This was a debugging comment I forgott to remove. I used it to remember where to change the type.